src/license_expression/__init__.py

```diff
@@ -599,6 +599,35 @@ def simple_tokenizer(self, expression):
                         sym = LicenseSymbol(key=sym_or_op)
                     yield Token(start, end, sym_or_op, sym)
 
+    def dedup(self, expression):
+        """
+        Return a de-dupulicated expression
+        """
+        l = Licensing()
+        exp = l.parse(expression)
+        operator = vars(exp)['operator']
+        dedup_expression = ''
+        expression_list = []
+        for arg in vars(exp)['args']:
```

Comment:
Same comment as above IMHO this si simpler:
```suggestion
        for arg in exp.args:
```